windows/Dockerfile.build

```diff
@@ -0,0 +1,18 @@
+FROM microsoft/windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+RUN Install-WindowsFeature NET-Framework-45-Core
```

Comment:
@StefanScherer it's listed as dependency [here](https://chocolatey.org/packages/vcbuildtools) and I don't think it worked without - let me check